### PR TITLE
Add crescent bot example

### PR DESCRIPTION
**Pull request by Lunarmagpie**
Docs mostly stolen from the other two bots ofc

Note: Crescent does not have the `GUILD_ID` variable because Its not needed for this lib.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ hikari[speedups]~=2.0.0.dev101; python_version < "3.10"
 hikari~=2.0.0.dev101; python_version == "3.10"
 hikari-lightbulb~=2.0.0
 git+https://github.com/FasterSpeeding/Tanjun@c1cb6c4aa1b103cda92f6779753867102ceb12d9
+hikari-crescent>=0.3.3
```
Comment (parafoxia) [inline requirements.txt:5]: I would prefer if this had an upper limit. I tend to do 0.x versions as `~=0.3.3`, but if you think it'll be stable throughout 0.x, use `>=0.3.3,<1.0`. I really need to go update the others lmao. If you want to go for it, but if not no worries.

File: crescent_bot/__main__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import crescent
+
+
+def create_bot() -> crescent.Bot:
+    # Load the token from a secrets file you'll need to create yourself.
+    with open("./secrets/token") as f:
+        token = f.read().strip()
+
+    # Create the main bot instance with all intents.
+    bot = crescent.Bot(token, intents=hikari.Intents.ALL_UNPRIVILEGED)
```
Comment (parafoxia) [inline crescent_bot/__main__.py:14]: Iirc the other examples use ALL, including privileged. If I'm wrong just ignore this.

File: crescent_bot/__main__.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+if __name__ == "__main__":
+    # if os.name != "nt":
+        # uvloop is only available on UNIX systems, but instead of
+        # coding for the OS, we include this if statement to make life
+        # easier.
+        # import uvloop
+
+        # uvloop.install()
```
Comment (parafoxia) [inline crescent_bot/__main__.py:29]: Is this supposed to be commented out?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # An intro to hikari
 
-This repo provides some simple examples to get you started with [hikari](https://github.com/hikari-py/hikari). Contained in this repo are bots designed with both the [hikari-lightbulb](https://github.com/tandemdude/hikari-lightbulb) and [hikari-tanjun](https://github.com/FasterSpeeding/Tanjun) command handlers. This repo is designed with extra stability in mind, but if you want newer features, [check out this experimental repo](https://github.com/Carberra/hikari-testing).
+This repo provides some simple examples to get you started with [hikari](https://github.com/hikari-py/hikari). Contained in this repo are bots designed with both the [hikari-lightbulb](https://github.com/tandemdude/hikari-lightbulb), [hikari-tanjun](https://github.com/FasterSpeeding/Tanjun) and [hikari-crescent](https://github.com/magpie-dev/hikari-crescent/) command handlers. This repo is designed with extra stability in mind, but if you want newer features, [check out this experimental repo](https://github.com/Carberra/hikari-testing).
```
Comment (parafoxia) [inline README.md:3]:
```suggestion
This repo provides some simple examples to get you started with [hikari](https://github.com/hikari-py/hikari). Contained in this repo are bots designed with both the [hikari-lightbulb](https://github.com/tandemdude/hikari-lightbulb), [hikari-tanjun](https://github.com/FasterSpeeding/Tanjun), and [hikari-crescent](https://github.com/magpie-dev/hikari-crescent/) command handlers. This repo is designed with extra stability in mind, but if you want newer features, [check out this experimental repo](https://github.com/Carberra/hikari-testing).
```

I'm an Oxford comma boi d:

File: README.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ There is a little bit of setup you need to do before testing the bots.
 
 ### hikari-lightbulb
 
-This library sports a very discord.py-like syntax (when making message commands), making for an easier time migrating for those wishing to switch. It is less powerful than hikari-tanjun, but is the more stable of the two options at present.
+This library sports a very discord.py-like syntax (when making message commands), making for an easier time migrating for those wishing to switch. It is less powerful than hikari-tanjun, but is the more stable of the three options at present.
```
Comment (parafoxia) [inline README.md:27]: This would probably make more sense if we changed this to simply say it's the most stable, removing the Tanjun comparison.

**Review by parafoxia — COMMENTED** [review]
Leaving these as comments as most changes I've mentioned may be a product of me messing something up before and not remembering lmao. Mostly looks good though.

File: crescent_bot/__main__.py (hunk 3 above)
Comment (Lunarmagpie) [reply crescent_bot/__main__.py:29]: oh oops

File: requirements.txt (hunk 1 above)
Comment (Lunarmagpie) [reply requirements.txt:5]: I don't know those libs at all so I don't want to touch them

File: README.md (hunk 5 above)
Comment (Lunarmagpie) [reply README.md:27]: will do

**Review by parafoxia — APPROVED** [review]

**Pull request merged**